### DISCUSSION TITLE
怎么指定gpu运行？怎么调整batchsize？

### DISCUSSION THREAD

**Discussion by zhejia** [PaddlePaddle/PaddleOCR#13862] [Q&A]
怎么指定gpu运行？怎么调整batchsize？

**Comment by UserWangZz** [comment]
https://paddlepaddle.github.io/PaddleOCR/index.html
请根据文档查询

**Reply by mayun123** [reply]
您好，没找到相关信息指定gpu进行训练

**Reply by UserWangZz** [reply]
在执行训练推理时添加下面代码即可   CUDA_VISIBLE_DEVICES=(GPU id) python train.py
